src/index.ts

```diff
@@ -846,18 +846,6 @@ export class Bigtable {
     }
 
     function makeRequestStream() {
-      const retryRequestOptions = Object.assign(
-        {
-          currentRetryAttempt: 0,
-          noResponseRetries: 0,
-          objectMode: true,
-        },
-        config.retryOpts
-      );
-
-      config.gaxOpts = Object.assign(config.gaxOpts || {}, {
-        retryRequestOptions,
-      });
```

Comment:
As noted in https://b.corp.google.com/issues/333878561#comment3, appending currentRetryAttempt, noResponseRetries and objectMode adds no value so we don't append these parameters anymore. In gax, the `retry` and `retryRequestOptions` properties cannot both be set and since using new gax streaming retries requires the `retry` property to be set, we must not set `retryRequestOptions` in the request anymore.